src/trials.jl

```diff
@@ -279,6 +283,10 @@ function prettymemory(b)
     return string(@sprintf("%.2f", value), " ", units)
 end
 
+function prettycount(n::Integer)
```

Comment:
Add a comment above this? Took me a sec to figure out what this did.

Reply:
Done, and tests.